### ISSUE TITLE
add machinelearningforkids.co.uk scratch exenstions

### ISSUE THREAD

**Issue by ikkqqstudio** [PenguinMod/PenguinMod-ExtensionsGallery#227]
add scratch exenstions form https://machinelearningforkids.co.uk/scratch/
there are missing exenstions form Scratch 3.0 GUI, PenguinMod never noticed today
screenshots:
![Screenshot 2024-08-16 18 59 01](https://github.com/user-attachments/assets/f9060ef1-b8c6-460f-805c-d7e4e508a614)
![Screenshot 2024-08-16 18 59 06](https://github.com/user-attachments/assets/5258e47c-5903-46f9-a8d6-2c7148b1af1c)
![Screenshot 2024-08-16 18 59 41](https://github.com/user-attachments/assets/4d428936-fbbc-40f4-aee7-7fffccbc0ed5)
![Screenshot 2024-08-16 18 59 46](https://github.com/user-attachments/assets/5b5b28a0-ff43-44b3-8cf4-06ec840982a1)

**Comment by SharkPool-SP** [comment]
half of them already exist under different names

**Comment by ikkqqstudio** [comment]
> half of them already exist under different names

what about Wikipedia and spotify, Raspberry Pi Pico (didn't own Raspberry Pi Pico)?

**Comment by TheShovel** [comment]
would be fun making an extension for raspberry pis. Might try doing that

**Comment by cicerorph** [comment]
> > half of them already exist under different names
> 
> what about most useful Exenstions for Scratch GUI such as Wikipedia and spotify, Raspberry Pi Pico (didn't own Raspberry Pi Pico), Weather, Face detection, Question Answering?

Wikipedia can be found on https://rubyteam.tech/gallery with the name PenguinPedia
Weather can be found on https://extensions.penguinmod.com/ with the name TurboWeather
Spotify can be found on SharkPool's Extension gallery: https://sharkpools-extensions.vercel.app/

**Comment by ikkqqstudio** [comment]
> > > half of them already exist under different names
> > 
> > 
> > what about most useful Exenstions for Scratch GUI such as Wikipedia and spotify, Raspberry Pi Pico (didn't own Raspberry Pi Pico), Weather, Face detection, Question Answering?
> 
> Wikipedia can be found on https://rubyteam.tech/gallery with the name PenguinPedia Weather can be found on https://extensions.penguinmod.com/ with the name TurboWeather Spotify can be found on SharkPool's Extension gallery: https://sharkpools-extensions.vercel.app/

thanks, 3 of them are existed

**Comment by cicerorph** [comment]
> > > > half of them already exist under different names
> > > 
> > > 
> > > what about most useful Exenstions for Scratch GUI such as Wikipedia and spotify, Raspberry Pi Pico (didn't own Raspberry Pi Pico), Weather, Face detection, Question Answering?
> > 
> > 
> > Wikipedia can be found on https://rubyteam.tech/gallery with the name PenguinPedia Weather can be found on https://extensions.penguinmod.com/ with the name TurboWeather Spotify can be found on SharkPool's Extension gallery: https://sharkpools-extensions.vercel.app/
> 
> thanks, 3 of them are existed

Btw for the Toxicity, im remaking an old extension called "Mouth Washer"

**Comment by ikkqqstudio** [comment]
> > > > > half of them already exist under different names
> > > > 
> > > > 
> > > > what about most useful Exenstions for Scratch GUI such as Wikipedia and spotify, Raspberry Pi Pico (didn't own Raspberry Pi Pico), Weather, Face detection, Question Answering?
> > > 
> > > 
> > > Wikipedia can be found on https://rubyteam.tech/gallery with the name PenguinPedia Weather can be found on https://extensions.penguinmod.com/ with the name TurboWeather Spotify can be found on SharkPool's Extension gallery: https://sharkpools-extensions.vercel.app/
> > 
> > 
> > thanks, 3 of them are existed
> 
> Btw for the Toxicity, im remaking an old extension called "Mouth Washer"

why is it called Mouth Washer?

**Comment by cicerorph** [comment]
Because it removes swear words, aka "removes" bad things from the "mouth"
of people

Em sex., 15 de nov. de 2024, 07:26, ikkqq ***@***.***>
escreveu:

**Comment by Monochromasity** [comment]
Please. If you want extensions from other sources to be added, add them yourself. If it's just you who wants the extensions, add them yourself and PR it. An issue just makes it complicated and I doubt anyone here will add them for you. Also, if you don't know how to add the extensions, just research it

**Comment by ikkqqstudio** [comment]
> Please. If you want extensions from other sources to be added, add them yourself. If it's just you who wants the extensions, add them yourself and PR it. An issue just makes it complicated and I doubt anyone here will add them for you. Also, if you don't know how to add the extensions, just research it

but It didn't work

**Comment by 1TSnakers** [comment]
If it is the TTS extension you want, there is an extension on the [PenguinMod Extra Extensions](https://extensions.penguinmod.com/). and then some of these extensions (like the books), can be "made" with the HTTP extension on the [TurboWarp Extension Gallery](https://extensions.turbowarp.org/) and the API they use.

**Comment by Ianyourgod** [comment]
> Please. If you want extensions from other sources to be added, add them yourself. If it's just you who wants the extensions, add them yourself and PR it. An issue just makes it complicated and I doubt anyone here will add them for you. Also, if you don't know how to add the extensions, just research it

I agree, but theres always the problem of licensing, so this isnt always the correct solution. Sometimes you just have to make one yourself.
